### PR TITLE
CIV-5708 - civil-service change from frozen lockfile to immutable for Yarn V3 compatability

### PR DESCRIPTION
**Pull request by mfallonhmcts**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: Jenkinsfile_CNP [hunk 1]
```diff
@@ -8,7 +8,7 @@ import uk.gov.hmcts.contino.GithubAPI
 def type = "java"
 def product = "civil"
 def component = "service"
-def ccdBranch = "master"
+def ccdBranch = "CIV-5708-second-PR"
```
Comment (filippo-possenti-version1) [inline Jenkinsfile_CNP:11]: This is temporary for the purpose of testing this PR, I assume.

Don't forget to remove this change once you're finished.

File: build.gradle [hunk 2]
```diff
@@ -213,7 +213,7 @@ task contract(type: Test) {
 }
 
 task pullTestAsset(type: Exec, description: 'Installs Yarn dependencies.') {
-  commandLine './bin/pull-latest-ccd-files.sh', 'master'
+  commandLine './bin/pull-latest-ccd-files.sh', 'CIV-5708-second-PR'
```
Comment (filippo-possenti-version1) [inline build.gradle:216]: This is temporary for the purpose of testing this PR, I assume.

Don't forget to remove this change once you're finished.

File: Jenkinsfile_CNP (hunk 1 above)
Comment (mfallonhmcts) [reply Jenkinsfile_CNP]: Thanks! Now removed and reverted to master

File: build.gradle (hunk 2 above)
Comment (mfallonhmcts) [reply build.gradle]: Thanks! Now removed and reverted to master

**Review by sabahirfan — APPROVED** [review]

**Review by MMNycz — APPROVED** [review]

**Review by asthamalviya — APPROVED** [review]

**Pull request merged**